types/map4dsdk/index.d.ts

```diff
@@ -0,0 +1,187 @@
+// Type definitions for map4dsdk 1.0
+// Project: https://github.com/iotlinkadmin/map4d-web-sdk (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Sua Le <https://github.com/sua8051>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'map4d' {
```

Comment:
Please use `declare namespace map4d` instead, following [this template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-d-ts.html).